dash/dash-renderer/src/utils/clientsideFunctions.ts

```diff
@@ -0,0 +1,27 @@
+import {updateProps, notifyObservers} from '../actions/index';
+import {getPath} from '../actions/paths';
+
+const set_props = (id: string | object, props: {[k: string]: any}) => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    const ds = (window.dash_stores = window.dash_stores || []);
```

Comment:
My TypeScript skills are pretty basic, but can't we avoid these comments (3 in this file and `store.ts`) with a few well-placed `as any` or some such? @T4rk1n you're better at this than I am, can you comment?